hotel-management/src/constants/routePath.ts

```diff
@@ -0,0 +1,12 @@
+const ROUTE_PATH = {
+  ERROR_PAGE: '*',
+  HOME_PAGE: '/',
+  ROOM_DETAILS_PAGE: '/room/:id',
+  CART_PAGE: '/cart',
+  LOGIN_PAGE: '/login',
+  REGISTER_PAGE: '/register',
+  ADD_ROOM_PAGE: '/add-room',
+  EDIT_ROOM_PAGE: '/edit-room',
+} as const;
+
+export default ROUTE_PATH;
```

Comment:
Why export default, just simply export named the `ROUTE_PATH` at line 1, if there was any rule against this disable it